src/main/java/picocli/AutoComplete.java

```diff
@@ -200,22 +200,22 @@ private boolean checkExists(final File file) {
     }
 
     /**
-     * Command that generates a Bash/ZSH completion script for its parent command (which should be the top-level command).
+     * Command that generates a Bash/ZSH completion script for its top-level command.
      * <p>
      * This class can be used as a subcommand for the top-level command in your application.
      * Users can then install completion for the top-level command by running the following command:
      * </p><pre>
-     * source &lt;(top-level-command generate-completion)
+     * source &lt;(top-level-command [sub-commands] generate-completion)
```

Comment:
Instead of `[sub-commands]`, let's use singular `[sub-command]`, or `[sub-command [sub-sub-command]...]` if we want to be very verbose.